http/src/request/channel/invite/create_invite.rs

```diff
@@ -62,19 +72,29 @@ impl<'a> CreateInvite<'a> {
     ///
     /// If no age is specified, Discord sets the age to 86400 seconds, or 24 hours.
     /// Set to 0 to never expire.
-    pub fn max_age(mut self, max_age: u64) -> Self {
+    pub fn max_age(mut self, max_age: u64) -> Result<Self, CreateInviteError> {
+        if max_age > 604_800 {
+            return Err(CreateInviteError::MaxAgeInvalid(
+                "max_age should be between 0 and 604800.".to_string(),
+            ));
+        }
```

Comment:
We place validation functions in [this file](https://www.github.com/twilight-rs/twilight/tree/trunk/http%2Fsrc%2Frequest%2Fvalidate.rs), so they should be moved there.